src/main/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByState.java

```diff
@@ -89,7 +89,8 @@ public static void validateOptions(Options options) {
 
   }
 
-  private static final String VARIANT_FIELDS = "variants(start,calls(genotype,callSetName))";
+  private static final String VARIANT_FIELDS =
+      "variants(alternateBases,calls(callSetName,genotype),end,referenceBases,start)";
```

Comment:
@pgrosu Please don't include code in your comments.  See https://github.com/googlegenomics/dataflow-java/blob/master/CONTRIBUTING.rst#contributor-license-agreements  Thank you!